src/test/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("The "+ getName() + " began its work.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("The "+ getName() + "has finished its work.");
```

Comment:
Similar to the 'doWork' method, the message in 'stopWork' should be more informative and indicate that an Excavator has finished its work. Also, there should be a space before 'has finished its work.' to separate the words properly.